app/classes/query/modules/sequence.rb

```diff
@@ -131,6 +131,12 @@ def first
     new_self
   end
 
+  def first_id
+    return nil unless result_ids.length
+
+    result_ids.first
```

Comment:
Is this method dead code? The only call I could find was QueryTest, line 428
If it's not dead, will this work:

```suggestion
    result_ids.&first
```

Reply:
Thanks. It may be dead code, but it's new code not old.

I wrote new `prev_id` and `next_id` methods and am using them in the `show_prev_next` UI and elsewhere, and i figured I'd need a `first_id` and `last_id` method to tell if we're at the first or last records. But I'm able to get it another way. 

I'll commit this suggestion but may end up deleting both.